### PR TITLE
Make Robottests work on Plone 5 - srly

### PR DESCRIPTION
**Pull request by jensens**
The very own tests were still on Plone 4.3. Now all is on 5.1 and should work on 5.0 and 5.2. Here the test matrix would need some enhancement. 

Before it was only half baken. And some keywords were broken. Tests too. I may have missed some case, but the tested cases are working now..

Support for Plone 4.3 was dropped.

### CODE REVIEW

**Comment by jensens** [comment]
If Saucelabs allows us to have a tunnel, tests are green. Saucelabs allows 5 parallel tunnels, and for some non-obvious reason, even if no tests are running, all 5 tunnels are used sometimes (or after some builds)

**Comment by datakurre** [comment]
@jensens I don’t believe we have benefitted from SauceLabs here much. It has been mostly as an example. If it gets on your way, just tear it off.

**Comment by jensens** [comment]
So, it works after given saucelab/travisci some relaxing time. Probably the tunnel is not closed properly. Anyway its green on Travis CI. Lets see if coredev goes green as well.

**Comment by jensens** [comment]
so, PR jobs on 5.2 are hanging unreleated b/c of https://github.com/plone/Products.CMFPlone/issues/2493

File: src/plone/app/robotframework/i18n.py [hunk 1]
```diff
@@ -1,25 +1,34 @@
...
         disableCSRFProtection()
-        portal = getSite()
-        portal_languages = getToolByName(portal, 'portal_languages')
         if language is None:
             language = os.environ.get('LANGUAGE') or 'en'
-        setattr(portal, 'language', language)
-        portal_languages.setDefaultLanguage(language)
+        try:
+            from Products.CMFPlone.interfaces import ILanguageSchema
+        except ImportError:
```
Comment (hvelarde) [inline src/plone/app/robotframework/i18n.py]: @jensens this code is for Plone 4 and should be removed.

Comment (jensens) [reply src/plone/app/robotframework/i18n.py]: Yes, I just made it work again, but, as stated above, I may have missed some case, or I may be too careful. Please file a new issue or provide a PR!

Comment (hvelarde) [reply src/plone/app/robotframework/i18n.py]: it depends on what you want; I think we can move on on these core packages leaving Plone 4 behind. let me know and I will add a PR with the clean up.

Comment (jensens) [reply src/plone/app/robotframework/i18n.py]: Yes, great! Plone 4.3 coredev already sits on the 1.2.x branch, so master targets already Plone 5.x only.

**Pull request merged**